include/mlir/Dialect/StandardOps/Ops.td

```diff
@@ -162,6 +162,13 @@ def AllocOp : Std_Op<"alloc"> {
     unsigned getNumSymbolicOperands() {
       return getNumOperands() - getType().getNumDynamicDims();
     }
+
+    /// Returns the symbolic operands (the ones in square brackets), which bind
+    /// to the symbols of the memref's layout map.
+    operand_range getSymbolicOperands() {
+      return {getOperation()->operand_begin() + getType().getNumDynamicDims(),
```

Comment:
I don't think getOperation()-> is necessary here.

Reply:
Fixed, thanks.